src/desktop/account.rs

```diff
@@ -34,29 +34,38 @@ struct UserInformationOptions {
     reason: Option<String>,
 }
 
-#[derive(Debug, DeserializeDict, Type)]
+#[derive(Debug, Default, SerializeDict, DeserializeDict, Type)]
 /// The response of a [`UserInformationRequest`] request.
 #[zvariant(signature = "dict")]
 pub struct UserInformation {
-    id: String,
-    name: String,
-    image: url::Url,
+    id: Option<String>,
```

Comment:
Why are those now nullable? I don't believe they are, are they?

Reply:
We needed to have Default there, I don't remember why. Url does not have default so it needs an option, maybe we can only wrap that one in option. In any case for me it makes more sense to pass None than `""`.

Reply:
When would we have a None here? I don't understand why Default is even needed here.

Reply:
This was reverted.